src/components/deposit/Congratulations.js

```diff
@@ -1,21 +1,30 @@
 import React from 'react'
 import { connect } from 'react-redux'
+import Lottie from 'react-lottie'
 
 import StatusIndicator from '../svgs/StatusIndicator'
-import TLogo from '../svgs/tlogo'
+import * as animationData from '../animation/tBTC-logo-animate.json'
```

Comment:
Feels like we should name this `tbtcLogoAnimationData` or similar, can be in a follow-up.

Reply:
Addressed in e0f1835